src/benchmarks/micro/libraries/System.Text.Json/Document/EnumerateArray.cs

```diff
@@ -63,7 +63,7 @@ public void EnumerateUsingIndexer()
             int arrayLength = _element.GetArrayLength();
             for (int j = 0; j < arrayLength; j++)
             {
-                JsonElement dummy = _element[j];
```

Comment:
Will this still work in 3.1 and 5.0? I don't remember what version of C# this is from.

Reply:
Yes, the discards were added in C# 7.0 ([doc](https://docs.microsoft.com/en-us/dotnet/csharp/discards)) and according to [this](https://stackoverflow.com/a/51729639/5852046) StackOverflow question, it's supported in .NET Core 1.0+. The CI leg for .NET 4.6.1 is also green, so it's 👍